custom_dc/custom_data.md

```diff
@@ -242,22 +242,24 @@ The `sources` section is optional. It encodes the sources and provenances associ
 
 To load custom data uploaded to Google Cloud, see instead [Pointing the local Data Commons site to the Cloud data](/custom_dc/data_cloud.html) for procedures.
 
+### Configure custom directories
+
+If you are using a directory other than `custom_dc/sample` to store your CSV files, edit the `env.list` file as follows:
```

Comment:
They would need to update the file for the sample as well.

Reply:
Oops, thanks for catching.